teuthology/task/install.py

```diff
@@ -65,7 +65,10 @@
     'python-ceph',
     'rbd-fuse',
 ]
-
+RHVERSION={}
+RHVERSION['ceph']={
+    '1.3.0':1    
+}
```

Comment:
@zmc this is to support for which versions are valid and also will be used in upgrades, list can do, but with dict i can hold additional info related related to each build and make use of it during upgrade.